css-three-column-layout/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container align-center text-align-center">
+    <div class="header">
+      <header class=" white-text-big"> Designed for life science.Optimized for SaMD and Digital Health.</header>
+    </div>
+    <div class="row align-center">
+      <div class="column">
```

Comment:
`column` what? How big is this column? `column` classes should be named after their size. This one is a third, so it should be `column-third`

Reply:
More descriptive! updated css/html from column to column-third.